tests/_internal/test_htm_file.py

```diff
@@ -35,5 +35,5 @@ def test_reads_online_id_from_url(self):
         self.assertEqual(read_online_id(file_contents), OnlineId(target_type="platform", slug="THIS-IS_a_SLUG_123"))
 
     def test_none_if_not_found(self):
-        file_contents = f"window.location.replace(https://os.mbed.com/about);"
```

Comment:
No need for this `f`.

Reply:
that is what the change was about. I removed the `f`

Reply:
I guess I'm not very good at reviewing.